packages/interface/src/GraphQLLanguageService.js

```diff
@@ -177,6 +179,22 @@ export class GraphQLLanguageService {
     return [];
   }
 
+  async getHoverInformation(
+    query: string,
+    position: Position,
+    filePath: Uri,
+  ): Promise<Hover.contents> {
+    const projectConfig = this._graphQLConfig.getConfigForFile(filePath);
+    const schema = await this._graphQLCache
```

Comment:
Same comment as the last PR here - could we defer fetching schema until we're absolutely in need of one?

Reply:
This is about as late as we can leave it – at this point the MessageProcessor has already that the hover position contains a graphql document